hw/ip/otp_ctrl/data/otp_ctrl_img_rma.hjson

```diff
@@ -257,7 +257,7 @@
                 },
                 {
                     name: "OWNER_SW_CFG_ROM_ALERT_DIGEST_RMA",
-                    value: "0x727ca8ae",
```

Comment:
this wasn't...manually updated was it..?

Reply:
Not yet, but soon we will be able to (thanks to @jon-flatley)